src/main/java/salesforce/ui/pages/HomePage.java

```diff
@@ -8,10 +8,6 @@
 public class HomePage extends BasePage {
     protected WebDriver driver;
 
-    public HomePage(WebDriver driver) {
-        super(driver);
-    }
-
     @Override
     protected void waitForPageLoaded() {
```

Comment:
This method is empty

Reply:
Yes, it was necessary to use homepagen to enter my feature, so it was not necessary to implement